quantuminspire/cli/command_list.py

```diff
@@ -335,6 +335,17 @@ def login(
     typer.echo(f"Using member ID {settings.auths[host].team_member_id}")
 
 
+@app.command("set-default-host")
+def set_default_host(
```

Comment:
Now that we have tha ability to set the `default_host`, should we also change the default value given to `host` when running `qi login` command? 
```
@app.command("login")
def login(
    host: str = typer.Argument(
        "https://api.qi2.quantum-inspire.com", help="The URL of the platform to which to connect"
    ),
) -> None:
```

Reply:
I think a user is expected to interact (mostly) with production env. So he should just be able to do a `poetry run qi login` without having to specify the url. Changing the default value will force the user to each time specify the url which may get annoying.

Reply:
Ah I see, in that case I think we can also implement this a bit differently here. 

We can drop the default value in the `login` command, because when we do  `settings = Settings()` , the `settings` variable already contains a default value (currently `default_host: Url = "https://staging.qi2.quantum-inspire.com"`). Why not change this to the production url (`"https://api.qi2.quantum-inspire.com"`). 

Then we can keep the `host` parameter in the login `method` optional. If it is provided, we use it to overwrite the `default_host`. If its not, we take the `default_host` from our newly created `settings` variable

Reply:
yes, I agree, changed it